po/ko.po

```diff
@@ -193,20 +196,22 @@ msgstr "참조"
 #: src/SUMMARY.md:53 src/references/shared.md:1
 #, fuzzy
 msgid "Shared References"
-msgstr "참조"
+msgstr "공유 Enum"
```

Comment:
The new translation is completely wrong here.

Reply:
This is a fuzzy entry, which means the msgmerge tried to match but it wasn't sure. Can you suggest a proper translation?